docs/resources/TestnetContracts.md

```diff
@@ -0,0 +1,90 @@
+# Deployment and Instantiation
+
+These are the steps to upload and instantiate a contract
+
+- Submit the upload proposal
+- Vote on the proposal, wait for it to pass
+- Submit proposal for instantiating the contract
+- Vote on the proposal, wait for it to pass
+- The contract is now instantiated and ready to be interacted with.
+
+See the list of codes that was uploaded to the testnet previously.
+```
+persistenceCore query wasm list-code --node https://rpc.testnet.persistence.one:443
+```
+
+You can set the `node` to the persistenceCore config and don't have to worry about passing that flag always
+
+```
+persistenceCore config node https://rpc.testnet.persistence.one:443
+```
+
+To upload the contract
+
+```
+RESP=$(persistenceCore tx gov submit-proposal wasm-store "<path/to/the/compiled/wasm>" \
+  --title "title" \
+  --description "description" \
+  --deposit 10000uxprt \
+  --run-as $VAL_ADDR \
```

Comment:
what is VAL addr, maybe we dont need this...
change the name to test-core-1 address
also chain id need to be defined.

Could you please review the PR urself.. it should all make sense, give it a through read

Reply:
$VAL_ADDR is the address of the validator. We have done it like this for all the contracts

Reply:
ya as a user this done be the variable we will be using.. and the address wont be a validator address

Reply:
change this to something more generic and mention what this value should be, basically an address with some test uxprt in there.
I dont mind if we use variables but mention how and what to set them